fava/templates/source.html

```diff
@@ -15,7 +15,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for source_file in api.source_files() %}
+                {% for source_file in api.source_files()|sort %}
```

Comment:
I think it's more of a representation thing, not really worth putting it in the API. I'm happy to change it though, if you still think it's better.